static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -25,6 +25,13 @@
         @include mq(desktop) {
             width: 100%;
         }
+
+        // hide * home links, which are duplicated on large screens
+        &:first-child {
```

Comment:
Do you think we should just add an extra class in the markup instead of relying on it always being the first item?